.vscode/launch.json

```diff
@@ -4,6 +4,13 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "type": "edge",
+      "request": "attach",
+      "name": "Attach to Edge",
+      "port": 2015,
```

Comment:
This is the default port that the configuration generated. There's no rhyme or reason for that particular port. If someone prefers another one, feel free to suggest the best port. 😉